src/hotspot/os/aix/porting_aix.cpp

```diff
@@ -244,21 +235,16 @@ bool AixSymbols::get_function_name (
       }
       p_name[i] = '\0';
 
-      // If it is a C++ name, try and demangle it using the Demangle interface (see demangle.h).
-#ifndef DISABLE_DEMANGLE
+      // If it is a C++ name, try and demangle it using the __cxa_demangle interface(see demangle.h).
       if (demangle) {
-        char* rest;
-        Name* const name = Demangle(p_name, rest);
-        if (name) {
-          const char* const demangled_name = name->Text();
-          if (demangled_name) {
-            strncpy(p_name, demangled_name, namelen-1);
-            p_name[namelen-1] = '\0';
+        int status;
+        char *demangled_name = abi::__cxa_demangle(p_name, nullptr, nullptr, &status);
+        if ((demangled_name != nullptr) && (status == 0)) {
+          strncpy(p_name, demangled_name, namelen-1);
+          p_name[namelen-1] = '\0';
+          ALLOW_C_FUNCTION(::free, ::free(demangled_name));
           }
```

Comment:
I suggest to fix the indentation.

Reply:
> I suggest to fix the indentation.

Fixed Indentation .Thank you @TheRealMDoerr .